core/src/main/codegen/templates/Parser.jj

```diff
@@ -3955,7 +3959,32 @@ SqlCreate SqlCreate() :
     }
 }
 </#if>
-
+<#if parser.execStatementParserMethods?size != 0>
+/**
+ * Parses an EXEC statement
+ */
+SqlNode SqlExec() :
+{
+    final Span s;
+	SqlNode macro;
+	SqlNode source;
+}
+{
+	/** Macro has the same domain as Table so TableRef() works */
+	<EXEC> macro = TableRef() { s = span(); }	
```

Comment:
I think this is fine but I looked at TableRef() and it seems to do way more than I need, is there an alternative?

Reply:
Agreed. We should try something a bit simpler like CompoundIdentifier().